reference/docs/conversion/sakai_11_mysql_conversion.sql

```diff
@@ -765,7 +765,7 @@ ALTER TABLE lti_content MODIFY title VARCHAR(1024);
 ALTER TABLE lti_content MODIFY pagetitle VARCHAR(1024);
 ALTER TABLE lti_content MODIFY consumerkey VARCHAR(1024);
 ALTER TABLE lti_content MODIFY secret VARCHAR(1024);
-alter table lti_content modify custom varchar(65536);
+ALTER TABLE lti_content MODIFY custom MEDIUMTEXT;
```

Comment:
It seems like his intention here was to use TEXT instead of MEDIUMTEXT. I see our databases use TEXT here